### ISSUE TITLE
example content using text with same id multiple times causing a11y tests to fail

### ISSUE THREAD

**Issue by jackahl** [kitconcept/kitconcept.intranet#199]
The example content for multiple content types used in the distro uses the same headlines "Text Heading H3" and "Text Heading H2" multiple times on the same page. This leads to errors in the a11y axe tests as those headlines always get the same id assigned through slate. 

URLs:
- https://plone-intranet.kitconcept.dev/features/content-types/page
- https://github.com/kitconcept/kitconcept.intranet/blob/db9c98a78148bd02798f2b34a3d7d29fc69816b5/frontend/cypress/tests/a11y/pageContent.cy.js#L22

Screenshot: 

<img width="3840" height="17388" alt="Image" src="https://github.com/user-attachments/assets/11a5e0b6-27b7-46c1-bd47-b19dc28ea785" />

There are 2 potential fixes for this:
- **A**: Disable the `duplicate-id-active` in the respective tests. (Currently implemented)
-  **B**: rename those duplicate headlines to be unique e.g. "Text Heading H3 A" 
-  **C**: patch slate to ensure ids in their blocks are unique

Both solutions seem fine by me. Question if we want to change the example content for the sake of not needing to change the a11y tests or we change the tests to allow us to keep (slightly )prettier example content.

**Comment by jackahl** [comment]
@tisto seems fine by me. I am not sure how big of an issue this is to grant a potentially technically complex solution. But let's continue with the changes to the example content for now

**Comment by tisto** [comment]
@jackahl, please make sure that you create an issue as a reminder for us that editors can create content that violates A11y guidelines with our editor. This is something that we will need to tackle (at least with a warning).

**Comment by jackahl** [comment]
@tisto I just had a discussion about the issue with @polyester @gforcada. We agreed that a technical solution to this problem is most likely not feasible. The 2 main ways we could (theoretically) solve this in theory are:
- A: somehow add a random string to the id to make it unique. For actual accessibility (not just technical compliance) this makes things actually worse for people relying on screenreaders, as they will get those random ids read out to them.
- B: build some sort of mechanism that warns editors about duplicate ids in the UI (maybe on save?). The problem here is that this has a high degree of technical complexity for debatable actual use. Technical depth here stems from the fact that all sorts of blocks, be it from core, addons, projects, etc. have headlines with ids (e.g. Slate, Grid, Listing, Maps, etc.). Blocks from third party addons could also use ids generated from user input, that is impossible to be taken into account. 

@polyester also argued that it is not realistic anyways to solve problems created by editors through the technical means in the CMS. E.g. Editors could also mix up the orders of headings, which we do not check. His suggestion was to rather recommend AXE browser tools to editors who have a desire to create fully accessible content.

I personally think that we should not spend too much resources on fixing problems that are actually only there to satisfy automated tests in CI, but affect actual users very little.

**Comment by tisto** [comment]
@jackahl please create this ticket in Volto so that we have a reference for future discussions. This was all I asked in the first place. Then we can close this ticket here (or create a new one in our GitLab).

**Comment by jackahl** [comment]
Split into issue for potential a11y checks in Volto: https://github.com/plone/volto/issues/7310
And fixing example content: https://gitlab.kitconcept.io/kitconcept/distribution-kitconcept-intranet/-/issues/145
